profile/detect.go

```diff
@@ -17,12 +17,31 @@
 package profile
 
 import (
+	"os"
+	"os/exec"
+	"path/filepath"
+
 	"github.com/buildpacks/libcnb"
 )
 
 func Detect(context libcnb.DetectContext) (libcnb.DetectResult, error) {
 	// NOTE: the logger is not passed into this function, that will likely be a change in libcnbv2
 
-	// TODO: implement detection logic
+	_, shErr := exec.LookPath("sh")
```

Comment:
```suggestion
	_, shErr := exec.LookPath("bash")
```

I think this should be bash. I can't remember the exact reason, but during the RFC discussion I'd mentioned making this use `sh` instead to potentially be more compatible and I believe @samj1912 mentioned a reason why it had to be `bash`.